CMakeLists.txt

```diff
@@ -309,10 +308,11 @@ endif()
 # Plugins
 set(LOVR 1)
 link_libraries(${LOVR_LUA})
-file(GLOB LOVR_PLUGINS plugins/*)
+file(GLOB LOVR_PLUGINS ${CMAKE_SOURCE_DIR}/plugins/*)
+message("Using these Lovr Plugins: ${LOVR_PLUGINS}")
```

Comment:
Personally I don't think this provides useful-enough information to warrant a message, but if it stays please spell as "LÖVR plugins".

Reply:
sorry, that was a leftover debug print. gone now.